website/Contribution/CONTRIBUTING.rst

```diff
@@ -94,6 +94,7 @@ A module is organized in a few directories:
 * `examples/`: external files
   `lib/`, ...
 * `models/`: model definitions
+* `readme/`: e.g. CONTRIBUTORS.rst, DESCRIPTION.rst, DEVELOP.rst, HISTORY.rst, INSTALL.rst, ROADMAP.rst, USAGE.rst
```

Comment:
This is giving examples, but it doesn't describe what the directory is for

Reply:
Would you prefer this?
* `readme/`: documentation

Reply:
I would recommend you provide both a description and examples, e.g.:

files used to build the module's README.rst file, e.g. DESCRIPTION.rst, CONTRIBUTORS.rst. To see all supported files, check [the module template](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)